docs/content/docs/test/Runner.md

```diff
@@ -17,48 +17,203 @@ It uses an internal TMS (test management system). We are working on support othe
 
 ## Getting started
 
-TBD
-
-## Configuration
-
-A configuration is done in gradle script:
+1. {{< expand "Apply the gradle plugin" >}}
+Add to your build.gradle.kts
 
 ```kotlin
 plugins {
     id("com.android.application")
     id("com.avito.android.instrumentation-tests")
 }
+```
+
+{{< /expand >}}
+
+2. {{< expand "Add common plugin configuration" >}}
 
+```kotlin
 extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
-    // all options goes here
+    // they are required for Avito app. We will make them optional in future.
+    reportApiUrl = "http://stub"
+    reportApiFallbackUrl = "http://stub"
+    reportViewerUrl = "http://stub"
+    registry = "registry"
+    sentryDsn = "http://stub-project@stub-host/0"
+    slackToken = "stub"
+    fileStorageUrl = "http://stub"
+
+    configurationsContainer.register("Local") {
+        tryToReRunOnTargetBranch = false
+
+        targetsContainer.register("api27") {
+            deviceName = "API27"
+
+            scheduling = SchedulingConfiguration().apply {
+                quota = QuotaConfiguration().apply {
+                    retryCount = 1
+                    minimumSuccessCount = 1
+                }
+
+                reservation = TestsBasedDevicesReservationConfiguration().apply {
+                    device = LocalEmulator.device(27)
+                    maximum = 1
+                    minimum = 1
+                    testsPerEmulator = 1
+                }
+            }
+        }
+    }
 }
-
 ```
 
-### Test APK
+{{< /expand >}}
 
-Plugin builds APKs on his own by default.\
-If for any reason you have to build APK externally, you can pass files manually:
+3. {{< expand "Run tests via gradle task" >}}
 
-```kotlin
-// optional
-applicationApk = "/path/to/app.apk"
-// optional
-testApplicationApk = "/path/to/test-app-debug-androidTest.apk"
+```shell script
+    ./gradlew :<projectPath>:instrumentation<ConfigurationName>           
+```
+In our case:
+```shell script
+    ./gradlew :<projectPath>:instrumentationLocal 
 ```
 
-### Tests filtering
-
-TBD
+{{< /expand >}}
 
 ## Examples
 
 Check out a configuration to run in `GradleInstrumentationPluginConfiguration` in the [test app](https://github.com/avito-tech/avito-android/blob/develop/subprojects/android-test/test-app/build.gradle.kts#L114)
 
-### How to run tests on local emulator
+## How to:
+
+Firstly you must apply a plugin TODO Getting started ref
```

Comment:
@eugene-krivobokov Help me, please. I don't get how to reference a title from the same page.

Reply:
https://avito-tech.github.io/avito-android/docs/contributing/docs/#links

`[title]({{< relref "#paragraph-to-link">}})`